docs/buildstrategies.md

```diff
@@ -106,6 +110,31 @@ kubectl apply -f samples/buildstrategy/kaniko/buildstrategy_kaniko_cr.yaml
 
 ---
 
+## Buildkit
+
+[Buildkit](https://github.com/moby/buildkit) is composed of the `buildctl` client and the `buildkitd` daemon. For the `buildkit` ClusterBuildStrategy, it runs on a [daemonless](https://github.com/moby/buildkit#daemonless) mode, where both client and ephemeral daemon run in a single container. In addition, it runs without privileges ( _[rootless](https://github.com/moby/buildkit/blob/master/docs/rootless.md)_ ).
+
+### Cache Exporters
+
+By default, the `buildkit` ClusterBuildStrategy will use caching to optimize the build times. When pushing an image to a registry, it will use the `inline` export cache, which pushes the image and cache together.
+
+### Known Limitations
+
+The `buildkit` ClusterBuildStrategy currently locks the following parameters:
+
+- A `Dockerfile` name needs to be `Dockerfile`, this is currently not configurable.
+- Exporter caches are enable by default, this is currently not configurable.
+
```

Comment:
```suggestion
- Even though the BuildKit build runs rootloss, it requires both [AppArmor](https://kubernetes.io/docs/tutorials/clusters/apparmor/) as well as [SecComp](https://kubernetes.io/docs/tutorials/clusters/seccomp/) to be disabled using the `unconfined` profile.

```

Reply:
good point, I think your suggestion have a typo, so let me manually enhance this via your comment.